frame/support/src/traits.rs

```diff
@@ -1733,11 +1733,10 @@ pub trait Instance: 'static {
 
 /// An instance of a storage.
 ///
-/// It is required the the couple `(PalletInfo::name<Pallet>(), STORAGE_PREFIX)` is unique.
+/// It is required the the couple `(pallet_prefix(), STORAGE_PREFIX)` is unique.
 /// Any storage with same couple will collide.
 pub trait StorageInstance {
-	type Pallet: 'static;
-	type PalletInfo: PalletInfo;
+	fn pallet_prefix() -> &'static str;
```

Comment:
Could you add some docs on what is what?

I'm a little bit confused.

The storage prefix is the one that the user gives? And the pallet prefix is the name of the pallet in the runtime?

Reply:
Yes storage_prefix is the name of the storage given by user, pallet prefix is the name of the pallet in the runtime, I updated doc